### ISSUE TITLE
[🚧 Bug]: category is displayed as void(0)

### ISSUE THREAD

**Issue by suprunchuk** [pheralb/svgl#490]
## **What is the issue?**  

On the website, a category is displayed as void(0) instead of its expected name or content. This issue occurs consistently across the affected section.
---

## Steps to Reproduce

1. Navigate to the website's main page.
2. Locate the section where categories are listed.
3. Observe the category labeled as void(0).

---

## Expected Behavior
Categories should be properly labeled or show their corresponding content.

## Actual behavior: 
The void(0) placeholder is displayed, rendering the category non-functional or unclear.

---

## Screenshots

![img-2024-12-31-09-44-42](https://github.com/user-attachments/assets/0ee43feb-cc28-4da4-9edb-491098c0c856)

---

## Additional Information

**Environment details:**

- OS: windows 11
- Browser (if applicable): Firefox 133.0.3

---

**Comment by suprunchuk** [comment]
https://svgl.app/directory/void(0)

**Comment by aryanprince** [comment]
@suprunchuk I don't think this is a bug, Void Zero ([link here](https://voidzero.dev/)) is Evan You's new devtools company. They maintain all the projects you see in your linked screenshot like Vite, Vitest, Rolldown and Oxc.

If anything, the category could be called "VoidZero" as mentioned in [Evan You's portfolio](https://evanyou.me/) since I believe "void(0)" is their wordmark logo. 

For example: the "Vercel" category only displays their company name as text and not their logomark/wordmark logo (the triangle logo), but the [VoidZero category on svgl.app](https://svgl.app/directory/void(0)) is it's wordmark (as text). But of course, this is a minor nitpick.

**Comment by suprunchuk** [comment]
> @suprunchuk I don't think this is a bug, Void Zero ([link here](https://voidzero.dev/)) is Evan You's new devtools company. They maintain all the projects you see in your linked screenshot like Vite, Vitest, Rolldown and Oxc.
> 
> If anything, the category could be called "VoidZero" as mentioned in [Evan You's portfolio](https://evanyou.me/) since I believe "void(0)" is their wordmark logo.
> 
> For example: the "Vercel" category only displays their company name as text and not their logomark/wordmark logo (the triangle logo), but the [VoidZero category on svgl.app](https://svgl.app/directory/void(0)) is it's wordmark (as text). But of course, this is a minor nitpick.

I got it. I understood what it is. so a minor quibble is still relevant if you use a direct link to this section.

**Comment by atleugim** [comment]
Hi, I've added a PR to resolve this: https://github.com/pheralb/svgl/pull/547

I think it's important to keep things consistently

**Comment by suprunchuk** [comment]
thx

**Comment by atleugim** [comment]
@suprunchuk   you can mark as completed, is merged in: https://github.com/pheralb/svgl/pull/547

You can see in: https://svgl.app/directory/voidzero
